thirdparty_builtin/CMakeLists.txt

```diff
@@ -5,6 +5,14 @@
 
 set(_blt_tpl_targets) # tracks names of enabled tpl targets
 
+if(ENABLE_GBENCHMARK OR ENABLE_GTEST OR ENABLE_GMOCK)
+  if("${CMAKE_CXX_STANDARD}" STREQUAL "" OR
+     "${CMAKE_CXX_STANDARD}" LESS 11)
+      message(WARNING "GoogleTest, GoogleMock, and GoogleBenchmark require greater than C++11. "
+                      "Set BLT_CXX_STD to 'c++11' or higher before loading BLT.")
```

Comment:
Should this disable GTest, etc. in this case?

Reply:
We had a few discussions on this.  The concern is you don't set it but the compiler naturally supports the standard you would want it to succeed. So throw a warning but don't fail.

Reply:
Sounds good. Thanks for the clarification.